sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -3853,7 +3853,7 @@ public async Task GetPropertiesAsync_IfTags_Failed()
                 e => Assert.AreEqual("ConditionNotMet", e.ErrorCode));
         }
 
-        [PlaybackOnly("Object Replication policies is only enabled on certain storage accounts")]
+        [Ignore("Object Replication policies is only enabled on certain storage accounts")]
```

Comment:
what's the plan to bring back this test?

Reply:
See above.